src/openrct2/drawing/ImageIndexType.h

```diff
@@ -0,0 +1,16 @@
+/*****************************************************************************
+ * Copyright (c) 2014-2024 OpenRCT2 developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#pragma once
+
+#include <cstdint>
+
+using ImageIndex = uint32_t;
+
+constexpr ImageIndex kSpriteIdNull = -1; // 0x7FFFF
```

Comment:
ImageIndex is `uint32_t`, so it can't be -1. Instead, use `std::limits<ImageIndex>::max()`. Don't forget to add `#include <limits>`.